fem/tmop.hpp

```diff
@@ -2105,7 +2109,9 @@ class TMOP_Integrator : public NonlinearFormIntegrator
    /// Parallel support for surface fitting to the zero level set of a function.
```

Comment:
Comment on the option to use interpolation vs the old approach (when evaluators are not specified). Mention that this option is only available in parallel for now.